src/components/AccordionPanel/AccordionPanel.tsx

```diff
@@ -82,14 +86,12 @@ export const AccordionPanel = ({
   }, [isActiveVar]);
 
   useEffect(() => {
-    setAriaControls(ariaControls || nanoid());
-    setAriaLabelledBy(ariaLabelledBy || nanoid());
```

Comment:
I guess this was to ensure `nanoid` only ran once? seems like a funky structure & the hook syntax makes more sense to me